docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +106,256 @@ To manage file storages:
+2. To retrieve the file URI, use the `toURI()` method:
 
    ```
-   tizen.filesystem.resolve('documents', onResolveSuccess, null, 'r');
+   function onsuccess(files) {
+       for (var i = 0; i < files.length; i++) {
+           // Display the file URI
+           console.log('File URI is: ' + toURI(files[i]));
+       }
+   }
+   function onerror(error) {
+       console.log(error);
+   }
+   tizen.filesystem.listDirectory('documents/subDir', onsuccess, onerror);
    ```
 
-   The `File` object (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/filesystem.html#File), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/filesystem.html#File), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/filesystem.html#File) applications) is returned in the success event handler.
+3. To retrieve the file content as a DOMString, binary data or to put it's contents to a `Blob` object use one of the following functions on `FileHandle` object opened in modes appropriate for reading ('r', 'rw').
 
-2. To retrieve a list of all the files and their directories located in a specified directory, use the `listFiles()` method of the `File` object:`function onResolveSuccess(dir) {    dir.listFiles(onsuccess);}`The method returns an array of `File` objects.
-
-3. To retrieve the file URI, use the `toURI()` method:
+   - First open a file using `openFile()` function with mode allowing to read it's contents.
+   ```
+   // Opening file for read - this code assumes that there is
+   // a file named 'file' in documents directory
+   var fileHandleRead = tizen.filesystem.openFile('documents/file', 'r');
+   console.log('File opened for reading');
+   ```
 
+   - Use readString() or readStringNonBlocking() to retrieve the file content as a DOMString.
    ```
-   function onsuccess(files) {
-       for (var i = 0; i < files.length; i++) {
-           /* Display the file name and URI */
-           console.log('File name is ' + files[i].name + ', URI is ' + files[i].toURI());
+   var fileContents = fileHandleRead.readString();
+   console.log('File contents: ' + fileContents);
    ```
 
-4. To retrieve the file content as a DOMString, use the `readAsText()` method.
+   - Use readData() or readDataNonBlocking() to retrieve the file content as a Uint8Array object (binary data).
+   ```
+   var fileContents = fileHandleRead.readData();
+   console.log('File binary contents in array:');
+   console.log(fileContents);
+   ```
 
-   The encoding input parameter of the method defines the format in which the file content is returned.
+   - Use readBlob() or readBlobNonBlocking() to retrieve the file content as a Blob object.
+   ```
```

Comment:
Would you please add 3 more spaces in front of the code, from line 215 to line 230?

This sample code box is sub item of `Use readBlob() or readBlobNonBlocking()~`  I think, so the layout should be fixed a little.

Reply:
Done.